sway/desktop/render.c

```diff
@@ -1040,8 +1052,16 @@ static void render_containers_tabbed(struct sway_output *output,
 
 	// Render surface and left/right/bottom borders
 	if (current->view) {
+		// Dim color
+		float* dim_color = config->dim_inactive_colors.unfocused;
+		if (is_urgent) {
+			dim_color = config->dim_inactive_colors.urgent;
+		}
+
 		struct decoration_data deco_data = {
 			.alpha = current->alpha,
+			.dim = current->current.focused ? 0.0f: 1.0f - config->dim_inactive,
+			.dim_color = dim_color,
```

Comment:
Use the `? :` conditional operator here as well to clean things up using the `urgent` var